utils/check_copies.py

```diff
@@ -29,15 +28,6 @@
 REPO_PATH = "."
 
 
-# This is to make sure the diffusers module imported is the one in the repo.
-spec = importlib.util.spec_from_file_location(
-    "diffusers",
-    os.path.join(DIFFUSERS_PATH, "__init__.py"),
-    submodule_search_locations=[DIFFUSERS_PATH],
-)
-diffusers_module = spec.loader.load_module()
-
-
```

Comment:
@patrickvonplaten This section is causing the failing fast tests for models and schedulers. `diffusers_module` doesn't appear to be used anywhere. So I removed the snippet. 

LMK if there's something I've missed here.

Reply:
That works!